translations/README-uz.md

```diff
@@ -1804,11 +1805,11 @@ Hech qachon tugatgan bo'lmaymiz.
 
 ---
 
-## Additional Detail on Some Subjects
+## Ba'zi mavzular bo'yicha qo'shimcha ma'lumotlar
 
-    I added these to reinforce some ideas already presented above, but didn't want to include them
-    above because it's just too much. It's easy to overdo it on a subject.
-    You want to get hired in this century, right?
+    Men bularni Yuqorida keltirilgan ba'zi fikrlarni mustahkamlash uchun qo'shdim, lekin yuqorida qo'shishni xohlamagandim 
+    chunki juda ko'p resurslarni o'z ichiga oladi. Bularni mavzu asosida qaytadan ko'rib chiqish afzalroq.
```

Comment:
"It's easy to overdo it on a subject" - "Bitta mavzuda juda ko'p qolib ketish ham yaxshi emas" degan manoda aytilgan